packages/vector/test-providers.ts

```
@@ -1,74 +1,76 @@
+	const providers = process.argv.slice(2);
+
+	if (providers.length === 0) {
+		console.log("Usage: bun test-providers.ts [chroma|qdrant|pgvector]...");
+		console.log("Example: bun test-providers.ts qdrant pgvector");
+		return;
+	}
+
+	for (const provider of providers) {
+		const config = VectorProviderFactory.getProviderFromEnv();
+		config.provider = provider as any;
+		await testProvider(config);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)